### DISCUSSION TITLE
Panel toggle on header click

### DISCUSSION THREAD

**Discussion by stominmetal** [primefaces/community#1540] [PrimeVue]
Is it possible to make the Panel header clickable so that one can toggle the Panel just like the button that exists now. It also should be possible to open it both ways, on box click or on button click. I tried using templating, but it doesn't include the whole header, now I am going to try to think of a workaround.

**Comment by tugcekucukoglu** [comment]
I suggest having a look at Fieldset or Accordion components.

**Reply by stominmetal** [reply]
Accordion is an intresting solution to my problem, but I still think it won't be a bad feature for the Panel Component. Nevertheless, thanks for the response, I will look more into Accordio
